get-started/index.md

```diff
@@ -67,42 +68,41 @@ packageserver-8cccc99dd-g7lkh       1/1     Running   0          3m3s
 Use the search bar to find the **Red Hat build of Cryostat** catalog item.
 {% include howto_step.html
   summary="Select the Cryostat Operator and click the Install button"
-  image-name="cryostat-operatorhub-install.png"
+  image-name="`cryostat-operatorhub-install.png`"
```

Comment:
I don't know/think this works - why does the image name need to be quoted like this?

Reply:
Yes, this does not work.